geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -605,7 +605,7 @@ start server --name=value [--assign-buckets(=value)] [--bind-address=value]
 </tr>
 <tr class="odd">
 <td><span class="keyword parmname">\-\-server-bind-address</span></td>
-<td>IP address on which the Server will be bound.</td>
+<td>The IP address to which the server will be bound. Set this option in a multi-homed server environment to allow server communications on a distict address. Setting a value of the empty string (&quot;&quot;) binds to all addresses.</td>
```

Comment:
Change "distict" to "distinct"

Reply:
change "server communications" to "server-to-server communications"?